### ISSUE TITLE
Couple of variables in the zh-CN translation config file are not translated

### ISSUE THREAD

**Issue by raducoravu** [dita-ot/dita-ot#2294]
If you look in the zh_CN.xml there are variables like:

                <variable id="Related Links">Related Links</variable>
                <variable id="Cross-Reference">Cross-Reference to:</variable>            
                <variable id="Content-Reference">Content-Reference to:</variable>            
                <variable id="Untitled section">Untitled section.</variable>            
                <variable id="List item">List item.</variable>
                <variable id="Foot note">Footnote.</variable>

which are not translated. Should I try to talk to one of our users to provide a translation for them? Or maybe @robander has colleagues who might help...

**Comment by jelovirt** [comment]
This should probably go to 2.2.5?

**Comment by infotexture** [comment]
See #2131 for a related issue.

**Comment by robander** [comment]
We do not have text for those as none of them have been used previously in our output - those and a couple others are still in English in most of the new languages I've updated over the last several months. For better or worse - I was unable to get the current PDF code to use most of these. Based on my notes from last fall:
* Related Links - I used the similar common XHTML string "Related information" for all new languages.
* Cross-Reference, Content-Reference: I could not get these to appear, I think they are now unused
* Untitled section: this also appears unused
* List item: I think this was only used for cross-reference to an unordered list item, in which case I think it's better practice and better output to use link text
* Footnote: this is referenced in the PDF code but appears unused, all of my test footnotes correctly used the number or the callout.

**Comment by robander** [comment]
I compared the `zh_CN.xml` variables file with the others I've been adding based on English. I found a couple of missing strings (troubleshooting note path, "Notices"), so I plan to add those. I've updated a couple of translations where available. For the remainder, as with other languages, I've marked several strings as "UNUSED" or "TODO" (many of the TODO items are only used in special circumstances, such as with DRAFT output). About to submit a pull request with the updated strings and comments.
